### PR TITLE
📝 Add bug report template

### PR DESCRIPTION
**Pull request by ntut-rick**
please check https://github.com/ntut-rick/ptsd/issues/new?assignees=&labels=bug&projects=&template=bug_report.yaml 
and https://github.com/ntut-rick/ptsd/issues/1

If you have time to check diffs better visit https://github.com/ntut-rick/ptsd/blob/template/.github/ISSUE_TEMPLATE/bug_report.yaml

-  related: #70

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      placeholder: "macOS 11.5"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: "CMake version"
+      placeholder: "3.27.8"
+    validations:
+      required: true
+
```
Comment (Dragonfly911117) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:51]: More toolchain info would be nice.
Check [the msg here](https://discord.com/channels/1093814266630512681/1093938857772322886/1206121976351952926).

**Comment by NOOBDY** [comment]
I feel like `cmake --system-information` is too verbose, also it doesn't correctly reflect my configuration Linux. i.e. I set up my environment using LLVM toolchain
`CC=clang CXX=clang++ cmake -B build -G Ninja`
but `cmake --system-information` shows my default compiler, which is `cc` and `c++`

**Comment by ntut-rick** [comment]
> I feel like `cmake --system-information` is too verbose, also it doesn't correctly reflect my configuration on Linux. i.e. I set up my environment using LLVM toolchain `CC=clang CXX=clang++ cmake -B build -G Ninja` but `cmake --system-information` shows my default compiler, which is `cc` and `c++`

I have found an alternative plan, which is to request them to provide `build/CMakeCache.txt`.

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (ntut-rick) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:51]: test if i can put suggestion here
```suggestion
  - type: textarea
    attributes:
      label: "CMakeCache.txt"
      description: |
        Can found in the build dir:
        - Clion (`cmake-build-debug/CMakeCache.txt`)
        - Visual Studio (`out\build\x64-Debug\CMakeCache.txt`)
        - Others (`build/CMakeCache.txt`)
        please use `Attach File` do not copy paste whole file.
    validations:
      required: true

```

Comment (ntut-rick) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:51]: @NOOBDY please take a look

**Review by Dragonfly911117 — APPROVED** [review]

**Pull request merged**